<?xml version="1.0" encoding="UTF-8"?>
<launch>

    <arg name="stage_world_file" />
    <arg name="headless" />
    <arg name="output" />

    <arg name="stage_args" value="-g"   if="$(arg headless)" />
    <arg name="stage_args" value=""     unless="$(arg headless)" />

    <param name="/use_sim_time" value="true" />

    <node name="stage" pkg="stage_ros_pm" type="stageros" required="true" args="$(arg stage_args) $(arg stage_world_file)" output="$(arg output)" >
        <param name="base_watchdog_timeout" value="0.3" />
    </node>

</launch>
